python_ta/transforms/z3_visitor.py

```diff
@@ -0,0 +1,43 @@
+import astroid
+from astroid import Uninferable, nodes
+from astroid.transforms import TransformVisitor
+
+from ..contracts import parse_assertions
+from .ExprWrapper import ExprWrapper
+
+
+class Z3Visitor:
```

Comment:
Please add a docstring here. (This is especially important because this is the main entrypoint into your Z3 work!)